CHANGELOG.md

```diff
@@ -1,5 +1,80 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.15.4
+<!-- generated comparing v5.15.3..master -->
+_Jan 9, 2024_
+
+A big thanks to the 20 contributors who made this release possible.
+This release was mostly about 🐛 bug fixes and 📚 documentation improvements.
+
+### `@mui/material@5.15.4`
+
+- [Dialog] Refactor form submission in FormDialog component (#40470) @sai6855
+- [TextField][FormLabel][InputLabel][FormControl] Use exact children type to allow React children type augmentation (#38872) @nicegamer7
+- [Select] Add form submission regression test (#40176) @mj12albert
+- [ToggleButtonGroup] Support different elements under it (#40220) @Methuselah96
```

Comment:
This could be highlighted similarly to ButtonGroup in [v5.14.9](https://github.com/mui/material-ui/releases/tag/v5.14.9).

Suggested highlight:
"✨ Material UI's [ToggleButtonGroup component](https://mui.com/material-ui/react-toggle-button/) now supports various elements underneath by removing React.cloneElement usage (e.g., a Tooltip or a disabled Tooltip on ToggleButton) (#40220)"